### PR TITLE
[1LP][WIPTEST] fix Tag Wise Graph

### PR DESCRIPTION
**Pull request by digitronik**
Purpose or Intent
=================
{{pytest: cfme/tests/candu/test_host_graph.py::test_tagwise --use-provider vsphere65-nested -sqvvv}}

run added for testing only...

### CODE REVIEW

**Comment by john-dupuy** [comment]
@digitronik, what is the update on this fix? I noticed that these tests are still failing with `AttributeError` on the most recent build.

**Comment by digitronik** [comment]
@john-dupuy this is quite interesting error. I am not able to reproduce this every time. but I was waiting for my `wt.pfy` `LineChart` widget so I can go for some refactoring related to `C&U` graph with @nachandr.

**Comment by nachandr** [comment]
Re-ran the tests on PRT today.

Some of the RHV tests are failing because of a timeout.

        # wait, for specific vm tag data. It take time to reload metrics with specific vm tag.
        wait_for(lambda: "London" in graph.all_legends,
>       delay=30, timeout=2400, fail_func=refresh)

**Comment by digitronik** [comment]
> Some of the RHV tests are failing because of a timeout.
@nachandr one test I found which may run last and data collected `http://10.16.45.124/iLIEXlqR/artifacts/cfme/tests/candu/test_host_graph.py/test_tagwise[rhevm-host_network-Hourly-vm_tag]/filedump-exception_screenshot.png` I think `RHV` taking more time for data collection.

**Comment by digitronik** [comment]
@nachandr... `Attribute` error fixed.

other:
I tried lot but `rhv` time is to much. I would advice to block this provider for these test as historical data tempting not supported by `rhv`.

I added this pr for attribute error fix. I think better to merge this and then we will sort `rhv` problem. This PR pending from last 4 month. at least with this PR vsphere test going to fix.

File: cfme/tests/candu/test_host_graph.py [hunk 1]
```diff
@@ -233,18 +233,15 @@ def test_tagwise(provider, interval, graph_type, gp_by, candu_tag_vm, enable_can
...
-        provider.refresh_provider_relationships()
-        host.capture_historical_data()
         provider.browser.refresh()
         view.wait_displayed(timeout='20s')
         view.options.fill(data)
 
     # wait, for specific vm tag data. It take time to reload metrics with specific vm tag.
-    wait_for(lambda: "London" in graph.all_legends,
-    delay=120, timeout=1500, fail_func=refresh)
+    wait_for(lambda: "London" in graph.all_legends, delay=60, timeout=3000, fail_func=refresh)
```
Comment (mshriver) [inline cfme/tests/candu/test_host_graph.py]: In the PRT results, I'm seeing the activity spinner active on the page - is it possible this fail_func refresh is actually triggering it to re-load, and thus is never allowing the activity spinner to complete? 

Refresh is actually doing a fill, re-filling the form every iteration. Doesn't matter how long the timeout is if every time its re-trying the options fill and submit.

Comment (digitronik) [reply cfme/tests/candu/test_host_graph.py]: It was my bad that i forgot to remove rhv related my tried. I increased timeout for rhv test. 
Yup, after browser refresh we need to refill data. need to select tag.

**Comment by digitronik** [comment]
Closing this PR... I think we no more need this.

**Pull request closed without merging**